Sources/MessagesViewController.swift

```diff
@@ -171,8 +171,8 @@ extension MessagesViewController: UICollectionViewDataSource {
         let message = messagesDataSource.messageForItem(at: indexPath, in: messagesCollectionView)
 
         switch message.data {
-        case .text, .attributedText:
-            let cell = messagesCollectionView.dequeueReusableCell(TextMessageCell.self, for: indexPath)
+        case .text, .attributedText, .emoji:
+            let cell = collectionView.dequeueReusableCell(TextMessageCell.self, for: indexPath)
```

Comment:
this should be `messagesCollectionView` but maybe its because this branch is not current with `v0.9.0`